creator-node/src/FileProcessingQueue.js

```diff
@@ -21,6 +21,8 @@ function constructProcessKey (taskType, uuid) {
 
 class FileProcessingQueue {
   constructor () {
+    const { logContext } = {}
+    this.logStatus(logContext, `Connecting to Redis`)
```

Comment:
I think it would be more useful to just add a log after the bull queue constructor. so something like `this.logStatus(logContext, 'Successfully initialized the FileProcessingQueue')`

Reply:
/ similar structure for the transcode queue + other queues we have

Reply:
also nit - we can probably refactor `this.logStatus()` to look like
```
async logStatus (logContext, message)  -> async logStatus (message, logContext = {})
```

this way, we can call `logStatus()` like: `this.logStatus('Connecting to Redis')` and then if we have a log context, just pass it in as the 2nd param

Reply:
and same with `this.logError()`

Reply:
let's also use single quotes if we're not adding any variables to the string msg